### DISCUSSION TITLE
Can you cover Jax in your next Tutorial??

### DISCUSSION THREAD

**Discussion by Aniruddha120** [mrdbourke/pytorch-deep-learning#449] [General]
Hey Daniel ,
I hope you are doing great! Can you please cover a day-long tutorial for Jax? just like the pytorch, covering every potential application?

Can you please implement all ML algos and then deep learning algos using it?

Thanks ! stay safe!

**Comment by dmahdipour** [comment]
I appropriate it if you make it possible.
